code/stim/de_MakeDataset.m

```diff
@@ -13,6 +13,9 @@
     if ~exist('show_figs', 'var'), show_figs = true; end;
     if ~exist('force', 'var'), force = false; end;
 
+    if strcmp(expt, 'vanhateren')
+        opt = {opt{1}};
```

Comment:
Haha making this kind of change is gonna be sooo hard to track down. There's gotta be a way to do this...